types/utils.d.ts

```diff
@@ -0,0 +1 @@
+export function isSlimSkin(skinImage: HTMLImageElement): boolean;
```

Comment:
Parameter type should be `HTMLImageElement | HTMLCanvasElement`.

Reply:
Missing defs of `loadSkinToCanvas` and `loadCapeToCanvas`.

Reply:
@g-plane I don't think `HTMLCanvasElement` should be acceptable. It's only for internal use. The canvas is required to have the skin image drawn on it and needs preprocessing, which is complex. And we haven't exported interfaces for that.